util/cron/common-perf-cray-cs-hdr.bash

```diff
@@ -5,6 +5,21 @@
 export CHPL_LAUNCHER_PARTITION=clx24
 export CHPL_TARGET_CPU=none
 
+COMMON_DIR=/cray/css/users/chapelu
+if [ ! -d "$COMMON_DIR" ]; then
+  COMMON_DIR=/cy/users/chapelu
+fi
+
+SETUP_PYTHON=$COMMON_DIR/setup_python_nightly.bash
+if [ -f "$SETUP_PYTHON" ]; then
+  source $SETUP_PYTHON
+fi
+
+echo "Location of Python3"
+which python3
+echo "Python3 version"
+python3 --version
+
```

Comment:
This seems odd to me. For all other configs I think we're either using the system python or bringing in a newer version in a .bashrc so it's not clear to me why this config is different/special.

Reply:
I did this because the arkouda testing that runs on the same machine explicitly sets its python version using a script call in the cron .bash file.  So I was worried that adding a .bashrc file to that machine would lead to confusion at best, and potentially weird Python errors from both being in the path at worst

Reply:
The other arkouda jobs (single node and XC) are in a similar position and hasn't been a problem (generally speaking having multiple python versions in the PATH is fine and the latest one will take precedence.)

Reply:
Okay, I added the .bashrc file to that machine.  Closing

Reply:
FWIW if you think it'd be helpful -- printing out the python version after changing it for arkouda isn't a bad idea (I'm fine not doing it too, your call.)